common/src/main/java/com/sequenceiq/cloudbreak/common/metrics/type/MetricType.java

```diff
@@ -6,6 +6,7 @@
     VAULT_WRITE("vault.write"),
     VAULT_WRITE_FAILED("vault.write.failed"),
     VAULT_DELETE("vault.delete"),
+    HEARTBEAT_UPDATE_SUCCESS("heartbeat.update.success"),
```

Comment:
do we need any modification on the metrics in datadog ?

Reply:
do we have a default mapper for this Enum?

Reply:
- metrics modification is not needed, this new metric however can be used to calculate a success/fail ratio
- no default enum is needed in this case as the enum used only for micrometer metric publishing (which either happens or does not)